src/components/Board/Board.container.js

```diff
@@ -1120,6 +1125,14 @@ export class BoardContainer extends Component {
     }
   }
 
+  onRequestToRootBoard() {
```

Comment:
I don't _think_ we want to return to the default ("root") board here, we probably want to return to the top board that was last loaded. So for example, if I open a board B and I click multiple folders, the button should take me back to board B, not the default board that's in `/board/root`.

@eadraffan @martinbedouret

Reply:
I agree :) I will correct that